app/src/main/java/com/example/cm/ui/adapters/FriendRequestListAdapter.java

```diff
@@ -96,19 +102,94 @@ private void onUndoDelete(FriendRequest request, int position, Request.RequestSt
         return new FriendRequestListAdapter.FriendRequestViewHolder(binding);
     }
 
+    @RequiresApi(api = Build.VERSION_CODES.N)
     @Override
     public void onBindViewHolder(@NonNull FriendRequestViewHolder holder, int position) {
         FriendRequest request = mRequests.get(position).getValue();
 
-        String user = Objects.requireNonNull(request).getSenderName();
+        String currentUserId = new AuthRepository().getCurrentUser().getUid();
+
+        if (request != null) {
+            if (request.getReceiverId().equals(currentUserId)) {
+                setReceivedRequests(holder, request);
+            }
+            if (request.getSenderId().equals(currentUserId)) {
+                setSentRequests(holder, request);
+            }
+        }
+
+
+    }
+
+    @SuppressLint("SetTextI18n")
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    private void setSentRequests(FriendRequestViewHolder holder, FriendRequest request) {
+        String fullName = getFullName(request.getReceiverId());
+        String userName = getUserName(request.getReceiverId());
+        String date = request.getCreationTimeAgo();
+
+        String requestDescription = "";
+
+        Request.RequestState requestState = request.getState();
+
+        if (requestState == Request.RequestState.REQUEST_PENDING) {
+            requestDescription = "Du wartest auf eine Antwort.";
+        } else if (requestState == Request.RequestState.REQUEST_ACCEPTED) {
+            requestDescription = "Deine Freundschaftsanfrage wurde bestätigt.";
```

Comment:
Wording ungünstig, weil in App nur "Deine Freundschaftsanfrage wurde..." zu sehen ist, da Zeile zu kurz